### PR TITLE
feat: Add `CalculationResultCompleted` integration event contract

### PR DESCRIPTION
**Pull request by BjarkeMeier**
## References

* #782

### CODE REVIEW

File: source/dotnet/Contracts/ProcessResultCalculated.proto [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  Grouping grouping = 999;
+  
+  enum Grouping {
+    GRID_AREA = 0;
+    ENERGY_SUPPLIER = 1;
+    BALANCE_RESPONSIBLE_PARTY = 2;
+    ENERGY_SUPPLIER_PER_BALANCE_RESPONSIBLE_PARTY = 3;
+  }
+
+  optional string GroupingActorGln = 998;
```
Comment (krmoos) [inline source/dotnet/Contracts/ProcessResultCalculated.proto:46]: Jeg foreslår tydelig kommentering af i hvilke tilfælde dette felt benyttes og hvorfor det er optionelt.

**Review by krmoos — APPROVED** [review]

File: source/dotnet/Contracts/ProcessResultCalculated.proto (hunk 1 above)
Comment (BjarkeMeier) [reply source/dotnet/Contracts/ProcessResultCalculated.proto:46]: Agree. I'll be adding thorough documentation to every single type and property.

Comment (kft) [reply source/dotnet/Contracts/ProcessResultCalculated.proto]: Maybe "Gln" is too specific. Actor numbers can be both GLN and EIC.

Comment (BjarkeMeier) [reply source/dotnet/Contracts/ProcessResultCalculated.proto]: Closing this conversation as the name has been updated to "EnergySupplierGlnOrEic" with the implied pros and cons as discussed earlier today.

File: source/dotnet/Contracts/calculation_result_completed.proto [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+  
+  // 3 digit scale decimal value of the energy quantity.
+  DecimalValue quantity = 2;
+
+  /*
+   * Currently holds exactly one quality.
+   * The quality currently represents the value used in RSM-014 CIM XML messages.
+   * Eventually this will be changed to support eBIX as-well in the EDI domain.
+   */
+  repeated QuantityQuality quantity_qualities = 3;
```
Comment (Sondergaard) [inline source/dotnet/Contracts/calculation_result_completed.proto:191]: If it only holds one value for now, why not represent it that way? I know from your comments that the usage might change later. If it changes, then we can update the contract to reflect this change.

**Review by kft — DISMISSED** [review]

File: source/dotnet/Contracts/calculation_result_completed.proto (hunk 2 above)
Comment (BjarkeMeier) [reply source/dotnet/Contracts/calculation_result_completed.proto]: Thx. Fixed.

**Review by lasrinnil — APPROVED** [review]

**Pull request merged**